syntax = "proto3";

message LocationMessage {
    int32 id = 1;
    int32 person_id = 2;
    string longitude = 3;
    string latitude = 4;
    string creation_time = 5;
}

service LocationService {
    rpc Create(LocationMessage) returns (LocationMessage);
}